ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/navigation/NavigateToFileViewImpl.java

```diff
@@ -323,7 +337,15 @@ void handleKeyDown(KeyDownEvent event) {
         new Timer() {
           @Override
           public void run() {
-            delegate.onFileNameChanged(fileName.getText());
+            String fileName = NavigateToFileViewImpl.this.fileName.getText();
+            // Skip handling non letter characters.
+            if (nativeKeyCode == KEY_BACKSPACE
```

Comment:
We can check it before timer creation

Reply:
https://github.com/eclipse/che/blob/21e4f80341657048f1d6502792d1d642774d6377/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/navigation/NavigateToFileViewImpl.java#L335-L336
I am not sure that the file name field will be ready to give text.